releng/org.eclipse.passage.loc.aggregator/pom.xml

```diff
@@ -71,7 +71,8 @@
 		
 		<module>../../bundles/org.eclipse.passage.loc.yars.api</module>
 	  	<module>../../tests/org.eclipse.passage.loc.yars.api.tests</module>
-	
+
+		<module>../../bundles/org.eclipse.passage.loc.report.core</module>
```

Comment:
we need to put before the repository and product

Reply:
Done